src/platform/client/client_platform_linux.cpp

```diff
@@ -26,7 +26,8 @@ void mcp::open_multipass_shell(const QString& instance_name)
 {
     assert(!instance_name.isEmpty() && "Instance name cannot be empty");
     QProcess::startDetached(
-        "xterm", {"-title", instance_name, "-j", "-e", QString("multipass shell %1 || read").arg(instance_name)});
+        "x-terminal-emulator",
```

Comment:
Going through `x-terminal-emulator` caters for running out of snap, does it?

Reply:
No, but I didn't get around to deciphering what it does for this case. Dumping it in was a BFI approach to getting things working.